modules/labs-react/core/stories/stories_visualTesting.tsx

```diff
@@ -84,22 +84,22 @@ export const TypeAndSpace = () => {
         <span css={[type.body, type.variant.mono]}>Mono Text</span>
       </section>
       <section className="story">
-        <h2 style={type.h2}>Spacing</h2>
-        <Box m={spacing.m} p={spacing.m}>
+        <h2 style={type.h2}>SpaceNumspaceNumbers</h2>
+        <Box m={spaceNumbers.m} p={spaceNumbers.m}>
           medium margin and padding
         </Box>
-        <Box mx={spacing.m} my={spacing.m} px={spacing.m} py={spacing.m}>
+        <Box mx={spaceNumbers.m} my={spaceNumbers.m} px={spaceNumbers.m} py={spaceNumbers.m}>
           medium margin (x, y) and padding (x, y)
         </Box>
         <Box
-          mt={spacing.m}
-          mr={spacing.m}
-          mb={spacing.m}
-          ml={spacing.m}
-          pt={spacing.m}
-          pr={spacing.m}
-          pb={spacing.m}
-          pl={spacing.m}
+          mt={spaceNumbers.m}
+          mr={spaceNumbers.m}
+          mb={spaceNumbers.m}
+          ml={spaceNumbers.m}
+          pt={spaceNumbers.m}
+          pr={spaceNumbers.m}
+          pb={spaceNumbers.m}
+          pl={spaceNumbers.m}
```

Comment:
Same comment here as above. All of these were manually updated because of the existing `space` declaration on line 6.